rc/filetype/asciidoc.kak

```diff
@@ -33,6 +33,9 @@ add-highlighter shared/asciidoc/ regex (\A|\n\n)[^\n]+\n\^{2,}\h*$ 0:header
 add-highlighter shared/asciidoc/ regex (\A|\n\n)=\h+[^\n]+$ 0:title
 add-highlighter shared/asciidoc/ regex (\A|\n\n)={2,}\h+[^\n]+$ 0:header
 
+add-highlighter shared/asciidoc/ regex ^//[^\n]*$ 0:comment
+add-highlighter shared/asciidoc/ regex ^////.*?^////$ 0:comment
```

Comment:
The [spec](https://asciidoc.org/userguide.html#X26) doesn't mention the minimum amount of slash signs required to start a comment, but looking at the example I would assume it's more than three, and the line has to contain only slash signs. Do you have more info about that?

Additionally, the idiomatic way of handling that would be with a region. Added to the consideration above, it would look something like (untested):

```kak
add-highlighter shared/asciidoc/regions regions
add-highlighter shared/asciidoc/regions/comment_block region ^/{3,}$ ^/{3,}$ fill comment
```

Reply:
according to this https://github.com/asciidoc/asciidoc/blob/master/asciidoc.conf, it must be at least 4 slashes, and the line should only be slashes.

Reply:
https://github.com/asciidoc/asciidoc/blob/0beddc73873acb3724b3cb93999acf0ddba84fc5/asciidoc.conf#L304-L308

Got it.

The regex still has to be fixed though.

Reply:
Yep, I'll push up a fix later today.